source/Xunit.Gherkin.Quick/CoreModel/Feature/Discovery/FeatureDiscoveryModel.cs

```diff
@@ -34,9 +34,10 @@ public List<FeatureFile> Discover(Type featureClassType)
                 
             if (fileClassInfo.IsPattern) {
                 _featureFileRepository
-                    .GetFeatureFilePaths()
-                    .FindAll(f => fileClassInfo.MatchesFilePathPattern(f))

```

Comment:
Isn't this what you are trying to fix? MatchesFilePathPattern - this returns false but must return true. Am I missing something?